Gemfile

```diff
@@ -1,6 +1,11 @@
 source 'https://rubygems.org'
 gemspec
 
-gem 'rails', '~> 3.2.0'
+gem 'jruby-openssl', :platforms => :jruby
+gem 'money', '~> 5.0'
```

Comment:
This is kind of weird. The gemspec accepts any version < 7, but the test suite doesn't work with version 6. DO we want to update the gemspec?